Tests/js/views/assets/ez-view-tests.js

```diff
@@ -36,7 +36,92 @@ YUI.add('ez-view-tests', function (Y) {
                 view.get('subPlainView').get('active'),
                 "The active attribute of the subPlainView should be undefined"
             );
-        }
+        },
+    });
+
+    addEventsTest = new Y.Test.Case({
+        name: "eZ View addEvent tests",
+
+        _testEventsInDom: function () {
+            var link1 = this.view.get('container').one('.event1'),
+                link2 = this.view.get('container').one('.event2');
+
+
+            link1.simulateGesture('tap', Y.bind(function () {
```

Comment:
choosing `tap` makes the whole thing a bit complicated since you are forced to have asynchronous test (wait / resume), it would be easier to use a regular DOM event like click. You could then write:

```js
link1.simulate('click');
Assert.isTrue(
    this.view.get('event1Fired'),
    "click on .event1 should have been caught"
);
```